React/Views/RCTView.m

```diff
@@ -223,7 +205,7 @@ - (NSString *)accessibilityLabel
   if (label) {
     return label;
   }
-  return RCTRecursiveAccessibilityLabel(self);
+  return nil;
 }
```

Comment:
Come to think of it, I might be able to just remove this method entirely since all it's doing is calling the super now.

Reply:
Update: removed. This should actually make it behave better with code like this (which is what my code actually does):

```objective-c
if ([view respondsToSelector:@selector(accessibilityLabel)])
    return view.accessibilityLabel;
```